cves/CVE-2004-0811.yml

```diff
@@ -76,9 +81,9 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: false
```

Comment:
I think for the "answer" field you are supposed to give a description as to what the state of unit testing was at the time of the fix.

Answer field from Meneely's example: "From looking at the fix code it's clear that code was tested and they added a new test for fixing this vulnerability."

Reply:
Thanks, that makes sense.